### DISCUSSION TITLE
Can PYG support Knowledge graph?

### DISCUSSION THREAD

**Discussion by Kevin-shihello-world** [pyg-team/pytorch_geometric#5803] [Ideas]
As the tittle

**Comment by manuel-dileo** [comment]
Yes, you can use heterogeneous graph learning models to solve learning task on KGs: https://pytorch-geometric.readthedocs.io/en/latest/notes/heterogeneous.html
You can find also some heterogeneous dataset based on KGs (e.g. https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html#torch_geometric.datasets.HGBDataset)
